internal/watchdservice/watchdservice.go

```diff
@@ -289,7 +289,7 @@ func (s *WatchdService) Status(ctx context.Context) (status string, err error) {
 	}
 
 	for _, dir := range svcInfo.dirs {
-		statStr.WriteString(fmt.Sprintf(i18n.G("\n  - %s"), dir))
+		statStr.WriteString(fmt.Sprintf("\n  - %s", dir))
```

Comment:
I think we maybe should?

Again, it’s not ideal because:
- there is no comment markup to help the translator to know what %s is :p
- we are using here positional arguments, so you can’t inverse 2 args.

It’s probably not really helpful as of now for translators, but if the 2 above is fixed in a future version, having the string already marked for translation would help spotting them, wdyt?